main/src/cgeo/geocaching/models/Geocache.java

```diff
@@ -1869,16 +1869,15 @@ public boolean hasStaticMap() {
     }
 
     /**
-     * Add spoilers stored locally in <tt>/sdcard/GeocachePhotos</tt>. If a cache is named GC123ABC, the
-     * directory will be <tt>/sdcard/GeocachePhotos/C/B/GC123ABC/</tt>.
+     * Add spoilers stored locally in <tt>/sdcard/cgeo/GeocachePhotos</tt>. If a cache is named GC123ABC, the
+     * directory will be <tt>/sdcard/cgeo/GeocachePhotos/C/B/GC123ABC/</tt>.
```

Comment:
I tried to reread all threads on the subject, and it looks like this has no importance after all, so feel free to rename.